inference-engine/tests/functional/plugin/shared/src/single_layer_tests/mat_mul.cpp

```diff
@@ -17,20 +17,18 @@ std::string MatMulTest::getTestCaseName(const testing::TestParamInfo<MatMulLayer
     InferenceEngine::Precision netPrecision;
     InferenceEngine::Precision inPrc, outPrc;
     InferenceEngine::Layout inLayout;
-    InferenceEngine::SizeVector inputShape0;
-    InferenceEngine::SizeVector inputShape1;
-    bool transpose_a;
-    bool transpose_b;
+    ShapeRelatedParams shapeRelatedParams;
     ngraph::helpers::InputLayerType secondaryInputType;
     std::string targetDevice;
-    std::tie(netPrecision, inPrc, outPrc, inLayout, inputShape0, inputShape1, transpose_a, transpose_b, secondaryInputType, targetDevice) =
+    std::map<std::string, std::string> additionalConfig;
+    std::tie(shapeRelatedParams, netPrecision, inPrc, outPrc, inLayout, secondaryInputType, targetDevice, additionalConfig) =
         obj.param;
 
     std::ostringstream result;
-    result << "IS0=" << CommonTestUtils::vec2str(inputShape0) << "_";
-    result << "IS1=" << CommonTestUtils::vec2str(inputShape1) << "_";
-    result << "transpose_a=" << transpose_a << "_";
-    result << "transpose_b=" << transpose_b << "_";
+    result << "IS0=" << CommonTestUtils::vec2str(shapeRelatedParams.firstInputShape) << "_";
+    result << "IS1=" << CommonTestUtils::vec2str(shapeRelatedParams.secondInputShape) << "_";
+    result << "transpose_a=" << shapeRelatedParams.transposeA << "_";
+    result << "transpose_b=" << shapeRelatedParams.transposeB << "_";
     result << "secondaryInputType=" << secondaryInputType << "_";
     result << "netPRC=" << netPrecision.name() << "_";
     result << "inPRC=" << inPrc.name() << "_";
```

Comment:
Please, add `result << config`, because names are can the same

Reply:
Thanks! Done.